feat: add terms & conditions page

lcsfelix opened the pull request.

lcsfelix commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Create the terms and conditions screen](https://trello.com/c/Wa0lscuJ/153-create-the-terms-and-conditions-screen)

RafaelTaranto commented on new-lamassu-admin/src/pages/OperatorInfo/OperatorInfo.js:
```diff
@@ -28,7 +29,8 @@ const TERMS_CONDITIONS = 'Terms & Conditions'
 
 const OperatorInfo = () => {
-  const [selected, setSelected] = useState(CONTACT_INFORMATION)
+  // const [selected, setSelected] = useState(CONTACT_INFORMATION)
```
Why is this commented out?

RafaelTaranto commented on new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js:
```diff
@@ -0,0 +1,166 @@
+            screenTitle: Yup.string().max(50, 'Too long'),
+            acceptButtonText: Yup.string().max(15, 'Too long'),
+            cancelButtonText: Yup.string().max(15, 'Too long')
+          })}
```
Move this to their own objects.

RafaelTaranto commented on new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js:
```diff
@@ -0,0 +1,166 @@
+    save({ show: s })
+  }
+
+  const classes = useStyles()
```
Move this to the top of the Component. Generally hooks should be the first thing called within the component function.

lcsfelix committed:
feat: add terms and conditions page

feat: add modal preview

feat: remove preview

fix: increase space between switch and fields

joshmh commented:
After merging some other PRs this PR now has conflicts. @lcsfelix can you resolve them?
